src/metpy/calc/thermo.py

```diff
@@ -3266,6 +3266,55 @@ def lifted_index(pressure, temperature, parcel_profile):
     return lifted_index
 
 
+@exporter.export
+@preprocess_and_wrap()
+@check_units('[pressure]', '[temperature]', '[temperature]')
+def k_index(pressure, temperature, dewpoint):
+    """Calculate K Index from the pressure temperature and dewpoint.
+
+    K Index formula derived from [George1960]:
```

Comment:
Need an underscore at the end so that Sphinx picks it up as a reference:
```suggestion
    K Index formula derived from [George1960]_:
```